netlify.toml

```diff
@@ -35,3 +35,9 @@ HUGO_ENABLEGITINFO = "true"
   to = "https://adoring-borg-aad42e.netlify.com/:splat"
   status = 200
   force = true
+
+[[headers]]
+  for = "/*"
+  [headers.values]
+    X-Frame-Options = "deny"
+    Strict-Transport-Security = "max-age=300; includeSubDomains"
```

Comment:
These sound still quite permissive. I would expect:

```
X-Content-Type-Options = "nosniff"
```

And then `Context-Security-Policy` with `default-src`, `connect-src`, `frame-src`, `img-src`, `script-src` with "self".

Reply:
There is a ticket for actually reviewing the CSP header and set the correct values.

> And then Context-Security-Policy with default-src, connect-src, frame-src, img-src, script-src with "self".

for the `X-Content-Type-Options = "nosniff"` I don't know about this header, I want to investigate it more before adding it.